nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -226,26 +238,24 @@ def transcribe(
         Returns:
             A list of transcriptions (or raw log probabilities if logprobs is True) in the same order as paths2audio_files
         """
-        if paths2audio_files is None or len(paths2audio_files) == 0:
+        if audio is None or len(audio) == 0:
             return {}
 
         if return_hypotheses:
             logging.warning("return_hypotheses=True is currently not supported, returning text instead.")
 
-        if isinstance(paths2audio_files, list):
-            logging.info(f"Found paths2audio_files to be a list of {len(paths2audio_files)} items.")
+        if isinstance(audio, list):
+            logging.info(f"Found paths2audio_files to be a list of {len(audio)} items.")
```

Comment:
remove references to `paths2audio_files` now that it is changed to `audio`

Reply:
Ah docstring. We'll have to do a major fix later anyway